variables.tf

```diff
@@ -7,3 +7,4 @@ variable vpc_cidr {}
 variable private_cidr {}
 
 variable public_cidr {}
+
```

Comment:
This extra newline is not needed, please remove it

Reply:
Updated removed the extra line